spec/mocks/serializers/user_serializer.rb

```diff
@@ -11,4 +11,6 @@ class UserSerializer
   attribute :description do |object|
     object.name.nil? ? nil : "My name is #{object.name} and I have #{object.cars.count} cars"
   end
+  attribute :bank_account, if: ->(_object, options) { options[:show_bank_account] }
```

Comment:
Change the name to https://github.com/CaioPenhalver/highlighter/pull/2/files#diff-1f0a8b414aecd35f542b431015559467d2c0b764ffc47f3146a2b072ceca1409R21 and create context when the value should be shown